handlers/search.go

```diff
@@ -0,0 +1,59 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	zebedeeCli "github.com/ONSdigital/dp-api-clients-go/v2/zebedee"
+	"github.com/ONSdigital/dp-cookies/cookies"
+	"github.com/ONSdigital/dp-frontend-search-controller/cache"
+	"github.com/ONSdigital/dp-frontend-search-controller/config"
+	"github.com/ONSdigital/dp-frontend-search-controller/data"
+	"github.com/ONSdigital/dp-frontend-search-controller/mapper"
+	"github.com/ONSdigital/dp-frontend-search-controller/model"
+	dphandlers "github.com/ONSdigital/dp-net/v2/handlers"
+	core "github.com/ONSdigital/dp-renderer/v2/model"
+	searchModels "github.com/ONSdigital/dp-search-api/models"
+	"github.com/ONSdigital/dp-topic-api/models"
+)
+
+// Read Handler
+func (sh *SearchHandler) Search(cfg *config.Config, template string) http.HandlerFunc {
```

Comment:
template parameter is not used
```suggestion
func (sh *SearchHandler) Search(cfg *config.Config) http.HandlerFunc {
```